datamade/bga-payroll
# Abstract data year toggle to be generalizable across templates

## Description

This PR refactors the data year toggle to load an initial year of data and abstracts the endpoint and post-request functionality into arguments so it can be used for multiple types of entity pages.

## Testing instructions

- Run the application locally: `docker-compose up -d`
- View the index page and confirm the most recent year of data loads automatically, and toggling the data year updates the page appropriately.
